wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
-| A preview point for this map is not set. Consider setting one from the Timing menu. |  |  |
-| Two timing points exist at the same time! |  |  |
-| {0} out of {1} timing sections have a volume below 5%. |  |  |
-| More than 1/3 of the map is Kiai time. Consider reducing this. |  |  |
-| More than 1/2 of the TV Size map is Kiai time. Consider reducing this. |  |  |
-| Kiai isn't snapped! |  |  |
-| Kiai's end isn't snapped! |  |  |
-| Breaktime is not suggested for mania maps. | osu!mania maps only! | Insert notes where the break time occurs. |
-| <!-- Can't confirm -->Easy/Normal diff contains too many speed changes. |  |  |
-| Kiai is toggled very frequently! |  |  |
+| Kiai needs an end time point. | *Self-explanatory* | Add a new inherited timing point to serve as the kiai's endpoint. | |
+| A preview point for this map is not set. Consider setting one from the Timing menu. | *Self-explanatory* | Set out a preview point for the map by clicking `Timing` » `Set Current Position as Preview Point`. | |
+| Two timing points exist at the same time! | *Self-explanatory* | Remove one of the conflicting timing sections. | |
+| {0} out of {1} timing sections have a volume below 5%. | *Self-explanatory* | Set the hitsound volume of the quiet timing sections to 5% or higher. | |
+| More than 1/3 of the map is Kiai time. Consider reducing this. | *Self-explanatory* | Reduce the kiai usage throughout the map. | This warning can safely be ignored if the kiai usage in question is intended. |
+| More than 1/2 of the TV Size map is Kiai time. Consider reducing this. | *Self-explanatory* | Reduce the kiai usage throughout the map. | This warning can safely be ignored if the kiai usage in question is intended. |
+| Kiai isn't snapped! | The specified kiai's starting point isn't snapped onto the timeline. | Snap the kiai's starting point properly to its intended position on the timeline. |
+| Kiai's end isn't snapped! | The specified kiai's ending point isn't snapped onto the timeline. | Snap the kiai's ending point properly to its intended position on the timeline. |
+| Breaktime is not suggested for mania maps. | *Self-explanatory* | Map out the part where the break time occurs. | ![](/wiki/shared/mode/mania.png) osu!mania maps only. This warning can safely be ignored if the usage of the break time in question is intended, or if the section of the song is otherwise unmappable. |
+| <!-- Can't confirm -->Easy/Normal diff contains too many speed changes. | The Easy/Normal difficulty features a frequent amount of slider velocity changes, which is discouraged by the [ranking criteria](/wiki/Ranking_Criteria). | Reduce the number of slider velocity changes on the Easy/Normal difficulty in question. | |
+| <!-- Can't confirm -->Kiai is toggled very frequently! | *Self-explanatory* | Reduce the number of kiai occurrences throughout the map. | Also known as *kiai flashes*. This warning can safely be ignored if the kiai usage in question is intended. |
```

Comment:
If set the timing like this, we can get this message.

![image](https://user-images.githubusercontent.com/110881926/231488694-d3fea064-ae0f-4db9-89b7-390f171f61ba.png)

Reply:
yup but i'm not sure if there's anything worth changing here tho, since the solution is still the same (to reduce the number of kiai occurrences throughout the map) lol

Reply:
this will also occur if multiple timing points (uninherited or inherited) are present in a kiai even if they do not toggle such kiai, you see this a lot in maps with frequent slider velocity changes, could be useful in the note?